### ISSUE TITLE
Team sync is not started unless you link a room first

### ISSUE THREAD

**Issue by LudvigHz** [matrix-org/matrix-appservice-slack#695]
**Describe the bug**
Team sync won't run even if you have all the configuration correctly set up.
This is due to the team never being created in the database without some action being taken and the team upsert in the datastore. An example is when doing a link command.

**To Reproduce**
1. Set up and configure the bridge
2. Configure the slack app and connect to workspace
3. Restart the bridge and observe the logs
4. No teams are synced even though they are configured

**Expected behavior**
All teams should sync when configured properly without having to issue another action.

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04

**Additional info**
It should be possible to upsert without issuing a `link` command. Or at least this behavior should be documented if it's not immediately fixable.

**Comment by Half-Shot** [comment]
Yep, I think we need a command to add a team to the bridge without adding a room to enable the team syncing system to work. This should be another fairly easy thing to add.

**Comment by freelock** [comment]
so I'm stuck here trying to add a new team -- I've been using the --slack_bot_token for the team to link a room, but now trying to bridge to a new team, I don't have that token (and it's not in the teams table of the database, where I've grabbed it previously).

So how do we get there from here? Is it possible to link the room without the team setup? If so, how?

I can't find the bridge bot user on the Slack side. This bridge is actively in use in 5 other Slack teams, so I think our config is ok, just struggling to set up a new team...
